cmake/HPX_GeneratePackageUtils.cmake

```diff
@@ -243,7 +243,7 @@ function(hpx_generate_pkgconfig_from_target target template is_build)
     INPUT ${OUTPUT_DIR}${template}_${build_type}.pc.in
     )
   # Temporary (to deprecate gradually)
-  if ("${CMAKE_BUILD_TYPE}" MATCHES "Rel")
+  if ("${build_type}" MATCHES "^rel")
```

Comment:
Actually I didn't put the `^` on purpose, it was to match `MinSizeRel` and `RelWithDebInfo` :
https://cmake.org/cmake/help/v3.15/variable/CMAKE_BUILD_TYPE.html

Reply:
Excellent catch, thanks! Just lower case `rel` would be fine?